extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -255,6 +255,7 @@ define('ext.wikia.adEngine.adContext', [
 		);
 
 		context.opts.outstreamVideoFrequencyCapping = instantGlobals.wgAdDriverOutstreamVideoFrequencyCapping;
+		context.opts.porvataMoatTracking = geo.isProperGeo(instantGlobals.wgAdDriverPorvataMoatTrackingCountries);
```

Comment:
what do you think about `porvataMoatTrackingEnabled`?

Reply:
Damn, line will exceed 120 chars :D 
👍 will do